tests/integration/cairo1_hint_locals/call/call_test.cairo

```diff
@@ -112,41 +77,15 @@ fn test_call_cairo0_non_existing_entrypoint() {
 
 #[test]
 fn test_call_wrong_name() {
-    let class_hash = declare('simple').unwrap();
-    assert(class_hash != 0, 'class_hash != 0');
-
-    let prepare_result = prepare(class_hash, ArrayTrait::new()).unwrap();
-
-    assert(prepare_result.contract_address != 0, 'deployed contract_address != 0');
-    assert(prepare_result.class_hash != 0, 'deployed class_hash != 0');
-
-    let prepared_contract = PreparedContract {
-        contract_address: prepare_result.contract_address,
-        class_hash: prepare_result.class_hash,
-        constructor_calldata: prepare_result.constructor_calldata
-    };
-    let deployed_contract_address = deploy(prepared_contract).unwrap();
+    let deployed_contract_address = deploy_contract('simple', ArrayTrait::new()).unwrap();
```

Comment:
What's the difference between this test and `test_call_cairo0_non_existing_entrypoint`?

Reply:
This case uses Cairo 1 contract and `test_call_cairo0_non_existing_entrypoint` uses Cairo 0 contract.